### ISSUE TITLE
Scrollbar doesn't repaint when showing/hiding development console (Inspect) in Chrome.

### ISSUE THREAD

**Issue by micobarac** [MurhafSousli/ngx-scrollbar#69]
### Bug Report or Feature Request (mark with an `x`)
```
- [*] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?
MacOS Mohave

### Versions
```
Angular CLI: 7.0.2
Node: 10.12.0
OS: darwin x64
Angular: 7.0.0
... animations, cdk, common, compiler, compiler-cli, core, forms
... http, language-service, material, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.10.2
@angular-devkit/build-angular     0.10.2
@angular-devkit/build-optimizer   0.10.2
@angular-devkit/build-webpack     0.10.2
@angular-devkit/core              7.0.2
@angular-devkit/schematics        7.0.2
@angular/cli                      7.0.2
@angular/flex-layout              7.0.0-beta.19
@ngtools/webpack                  7.0.2
@schematics/angular               7.0.2
@schematics/update                0.10.2
rxjs                              6.3.3
typescript                        3.1.3
webpack                           4.19.1
```

### Repro steps
Scrollbar doesn't repaint (update height) when showing/hiding development console (Inspect) in Chrome. It refreshes only when I start scrolling. 
The same problem happens on windows resize.
### Desired functionality
Scrollbar should repaint when showing/hiding development console (Inspect) in Chrome and resizing window.

**Comment by MurhafSousli** [comment]
I am about to publish a new version, so stay tuned

**Comment by micobarac** [comment]
It also happens when I use div which expands on click... The scrollbar height stays the same.

**Comment by MurhafSousli** [comment]
@micobarac For opening the dev console or window resize, this is not part of this plugin.
You can update the scrollbar manually using `scrollbar.update()` function manually?

**Comment by MurhafSousli** [comment]
I just published v3.0.0, please try it out. closing this!
